modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/TypeScriptFetchClientCodegen.java

```diff
@@ -1109,7 +1115,7 @@ public String getItemsDataType() {
         public ExtendedCodegenProperty(CodegenProperty cp) {
             super();
 
-            this.openApiType = openApiType;
+            this.openApiType = cp.openApiType;
```

Comment:
that's cute.
luckily this property is never used in this generator. i mean - looks like it's almost never used in this repo at all :)